node_modules/nforce-tooling/index.js

```diff
@@ -0,0 +1,449 @@
+var _ = require('lodash');
```

Comment:
@kevinohara80 I have a PR pending with nforce-tooling, as well.  I was hoping to add some further granularity to the time each test takes to run using DMC, but the nforce-tooling repo appears abandoned and unfortunately the fields required for DMC to show the test times aren't retrieved by the current SOQL being executed.  I read the article you had linked in your .gitignore about including node modules with some interest, and had a feeling this would be something you asked about.  I'm unsure of what to do - I could of course re-do the PR to only fix the issue reported with the table declaration being null (when dependent classes are invalid) and leave the rest of these changes out of the PR.